docs/pages/2019_MARVEL_Psik_MaX/sections/verdi_cmdline.rst

```diff
@@ -365,9 +365,10 @@ You should be able to rotate the view with the right mouse button.
 Alternatively, especially if showing them interactively is too slow over SSH, you can export the content of a structure node in various popular formats such as ``xyz`` or ``xsf``.
 This is achieved by typing in the terminal:
 
-.. code:: console
+.. code:: bash
```

Comment:
this should remain "console" (used for uniform styling of the background)

Reply:
You might be aware, but this is how it looks with bash,

![console](https://user-images.githubusercontent.com/1621518/57978402-9b4a1d80-7a0d-11e9-80ec-29a5d5508bb7.png)

It's pretty clear that the first line is a comment and the second one you should use, conversely this is how it looks with bash:

![actual console](https://user-images.githubusercontent.com/1621518/57978412-c5034480-7a0d-11e9-982c-a279e8680d2c.png)

To my eyes the comment is hardly distinguishable, so I'm okay with 3 different options,

1. Sacrificing consistency for the sake of clarity and not make an update.
2. Sacrificing clarity for the sake of consistency and update it to `console`.
3. Getting consistency and clarity through work, and update all the consoles to bash, and make sure python code blocks look different enough.

After stating what's obvious to me, I'd appreciate you help me pick between the three equally valuable options.

Reply:
oh, sorry - I wasn't aware of this difference, thanks for pointing it out!
I think 3. is the right approach - I'll take care of this (but merge your PR first).